tchannel/event.py

```diff
@@ -60,6 +61,7 @@ def before_send_request(self, request):
                 ....
 
     """
+
```

Comment:
why the space?

Reply:
I use autopep8. It added it automatically